src/app/azureResource/sagas/setActiveAzureResourceSaga.ts

```diff
@@ -3,12 +3,18 @@
  * Licensed under the MIT License
  **********************************************************/
 import { Action } from 'typescript-fsa';
-import { put } from 'redux-saga/effects';
+import { call, put } from 'redux-saga/effects';
 import { AzureResource } from '../models/azureResource';
 import { clearDevicesAction } from '../../devices/deviceList/actions';
 import { clearModelDefinitionsAction } from '../../devices/deviceContent/actions';
+import { ACTIVE_CONNECTION_STRING } from '../../constants/browserStorage';
 
 export function* setActiveAzureResourceSaga(action: Action<AzureResource>) {
+    yield call(setActiveConnectionString, action.payload.connectionString);
     yield put(clearDevicesAction());
     yield put(clearModelDefinitionsAction());
 }
+
+export const setActiveConnectionString = (value: string): void => {
```

Comment:
One thing minor: let's put some //todo here to remind us, and aslo clear the doubts if others are working on this file, as this was designed with a different purpose